app/src/main/res/layout/activity_main.xml

```diff
@@ -1,19 +1,237 @@
 <?xml version="1.0" encoding="utf-8"?>
-<android.support.constraint.ConstraintLayout
-    xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     tools:context=".MainActivity">
 
-    <TextView
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:text="Hello World!"
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:orientation="vertical"
         app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintLeft_toLeftOf="parent"
-        app:layout_constraintRight_toRightOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_weight="@integer/weight_v_parent">
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="@dimen/dimen_results_size"
+            android:layout_gravity="center"
+            android:layout_weight="@integer/weight_v_pos_1"
+            android:orientation="vertical">
+
+            <TextView
+                android:id="@+id/text_results"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginLeft="@dimen/dimen_small_margin"
+                android:layout_marginTop="@dimen/dimen_small_margin"
+                android:layout_weight="@integer/weight_v_pos_1_1" />
+
+            <TextView
+                android:id="@+id/text_operations"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginLeft="@dimen/dimen_small_margin"
+                android:layout_weight="@integer/weight_v_pos_1_2" />
+        </LinearLayout>
+
+        <GridLayout
+            android:id="@+id/gridLayout"
```

Comment:
Try to name the xml ids with the same naming convention in all the project.  In this case, grid_layout

Reply:
Done

Reply:
Done